sdk/core/core-client-rest/package.json

```diff
@@ -80,6 +80,7 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/core-tracing": "^1.0.1",
     "@azure/core-util": "^1.0.0",
```

Comment:
Do we need all of these or are we just pulling directly from the ts-http-runtime?

Reply:
Checked through these. Looks like we can drop core-util, but we reference types from all the others so I don't think we can drop them unfortunately. Thanks for picking that up.